### PR TITLE
[RAPPS-DB] Add Cave Story

### PR DESCRIPTION
**Pull request by hope2586**
This adds Cave Story to the application database. Here's a screenshot of it running under ReactOS:

VirtualBox_ReactOS - Fresh_03_12_2022_15_10_54

### CODE REVIEW

**Comment by hope2586** [comment]
Any thoughts on this?

File: cavestory.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[Section]
+Name = Cave Story
+Version = 1.0.0.6
+License = Freeware
+Description = A Japanese freeware PC side-scrolling platformer game.
+Category = 4
+URLSite = https://www.cavestory.org/
+URLDownload = https://www.cavestory.org/downloads/cavestoryen.zip
```
Comment (learn-more) [inline cavestory.txt:8]: Could you please add the sha1 hash of the zip file?
That way we can see when the zip file is replaced with a version that is not tested.
Preferrably, link to a version that stays 'stable'

Comment (hope2586) [reply cavestory.txt:8]: It has been done. Do you want a link to a video of it?

**Review by learn-more — APPROVED** [review]

**Comment by hope2586** [comment]
Anyone else needed to approve it?

**Comment by learn-more** [comment]
Thanks for your contribution, and sorry it took a while to merge it.

**Comment by hope2586** [comment]
You're welcome.

**Pull request merged**